docs/snippets/dynamic.py

```diff
@@ -59,7 +66,7 @@ def create_attributes(parameters: dict[str, Any]) -> dict[str, Attribute]:
 @dataclass
 class TemperatureControllerHandler(AttrHandlerRW):
     command_name: str
-    update_period: float | None = 0.2
+    update_period: UpdatePeriod = 0.2
```

Comment:
Does the type hint have to be `UpdatePeriod` to make the type checker happy? It might be nicer to keep this union definition private.